packages/ckeditor5-dev-env/lib/translations/transifex-service-for-api-v3.0.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+const { transifexApi } = require( '@transifex/api' );
+const fetch = require( 'node-fetch' );
+
+const MAX_DOWNLOAD_ATTEMPTS = 15;
+const TIMEOUT_BETWEEN_DOWNLOAD_ATTEMPTS = 2;
+
+/**
+ * Promise wrappers of the Transifex API v3.0.
+ *
+ * @see https://docs.transifex.com/api-3-0/introduction-to-api-3-0 for API documentation.
+ */
+module.exports = {
+	init,
+	getProjectData,
+	getTranslations,
+	getResourceName,
+	getLanguageCode
```

Comment:
I want to have a consistent interface for this Transifex service: the Transifex service operates on resource instances and language instances, so let it provide methods for extracting the data from these instances, so that the consumer does not need to know about the internal data format. The `getResourceName()` function is used in the download script. The `getLanguageCode()` is used only internally in this service, but to keep it consistent with the other one, I would like to keep it as it is now - as one-line util. In addition, exporting this method allows it to be easily tested.